doc/intro.rst

```diff
@@ -26,7 +26,7 @@ Slim, Yii, Laravel, and Codeigniter — just to name a few.
 Prerequisites
 -------------
 
-Twig needs at least **PHP 5.4.0** to run.
+Twig needs at least **PHP 5.5.0** to run.
```

Comment:
hmm not sure about that. 5.4 can still be used with `version <= 1.42.2`

Reply:
That's what we did before when we bumped the min PHP version for 1.x, so LGTM